### PR TITLE
Remove unwanted file

### PR DESCRIPTION
**Pull request by digitalinfinity**
Testing the OS X builds in Jenkins

### CODE REVIEW

**Comment by digitalinfinity** [comment]
@dotnet-bot test this please

**Comment by digitalinfinity** [comment]
@dotnet-bot test this please

**Comment by digitalinfinity** [comment]
@dotnet-bot test this please

File: test/Error/rlexe.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (dilijev) [inline test/Error/rlexe.xml]: BOM change?

**Comment by digitalinfinity** [comment]
@dotnet-bot test this please

**Comment by digitalinfinity** [comment]
@dotnet-bot test this please

**Comment by dilijev** [comment]
@dotnet-bot 
test Windows arm_test please
test Windows x86_release please

@mmitche XAMLTaskFactory failures in
http://dotnet-ci.cloudapp.net/job/Microsoft_ChakraCore/job/master/job/arm_test_prtest/1816/
http://dotnet-ci.cloudapp.net/job/Microsoft_ChakraCore/job/master/job/x86_release_prtest/1822/

**Comment by mmitche** [comment]
@dilijev It's also possible for recent changes in your build affecting timing to make this more likely to happen.

**Comment by obastemur** [comment]
Guess this PR has already served its purpose

**Pull request closed without merging**